Documentation/ceph-pool-radosnamespace.md

```diff
@@ -0,0 +1,44 @@
+---
+title: RadosNamespace CRD
+weight: 3610
+indent: true
+---
+
+{% include_relative branch.liquid %}
+
+This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](quickstart.md)
+
+# CephBlockPoolRadosNamespace CRD
```

Comment:
How would a volume be provisioned inside the namespace? Does the storage class have some property? Or is csi integration not implemented yet?